src/components/SearchForm/SearchForm.jsx

```diff
@@ -0,0 +1,43 @@
+import React, { Component } from "react";
+import "./styles.css";
+
+const defaultValue = "";
```

Comment:
Please, use UPPER_CASE for constants

Reply:
Done (acb780a178648469d974bc88446542caddf0576e)